### DISCUSSION TITLE
Allow Only Reports To Specific Role, Disabling the Alerts

### DISCUSSION THREAD

**Discussion by codegeekjk** [apache/superset#15443] [General]
We want to provide only specific users access to the Reporting Feature of `Alerts and Reports`  but we don't want them to access alerts.

So is there any way via which we can enable Reports only and disabling the alerts.

**Comment by willbarrett** [comment]
Currently the two features are tied together. They share the same UI code and back-end, so it is not currently possible to enable one but disable another.

**Reply by codegeekjk** [reply]
Can you point me out if  from frontend  we can disable it, so we can patch our own fix.

**Reply by nytai** [reply]
Here are the routes https://github.com/apache/superset/blob/070f0b6cb2a1c3385bc2a7ed2448aa7eb1251666/superset-frontend/src/views/routes.tsx#L151

You might have to remove the button from the sub menu UI as well
